src/cryptoadvance/specter/services/service_encrypted_storage.py

```diff
@@ -98,6 +99,14 @@ def get_service_data(self, service_id: str) -> dict:
         return service_data
 
 
+class ServiceUnencryptedStorage(ServiceEncryptedStorage):
```

Comment:
Is that really enough? No "encrypted=False" etc.

Reply:
I've added some sanity-checking and a documentation for that class.